sqlite-db/src/test_events/rollover_completed.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "RolloverCompleted",
   "data": {
-    "funding_fee": { "fee": 100, "rate": 1.5 },
+    "funding_fee": { "fee": 100, "rate": 0.015 },
```

Comment:
This was actually a "bug" in the old tests but we have never seen it because we did not need it. Now, I parse the rate using `new` which will fail if the rate is above 1.